wpilibcExamples/src/main/cpp/examples/ElevatorSimulation/cpp/Robot.cpp

```diff
@@ -53,7 +62,8 @@ class Robot : public frc::TimedRobot {
   frc::DCMotor m_elevatorGearbox = frc::DCMotor::Vex775Pro(4);
 
   // Standard classes for controlling our elevator
-  frc2::PIDController m_controller{kElevatorKp, 0, 0};
+  frc2::PIDController m_controller{kElevatorKp, kElevatorKi, kElevatorKd};
+  frc::ElevatorFeedforward<units::meters> feedforward(kElevatorkS, kElevatorkG, kElevatorkV, kElevatorkA);
```

Comment:
The template argument needs to be removed.

kS, kG, kV, and kA need to be in the proper units.